### PR TITLE
Feature/chapter 1

### LINKED ISSUE #2: Chapter 1: Why version control
## Background

Write chapter that covers the following:

1. What benefits does version control have
2. Different ways of versioning your code manual -> git
3. Introduction to collaboration software inc git and vsts

## To do 

1. Create 01-whyversioncontrol.Rmd to make this chapter 2 in the bookdown.
2. Iterate guidance

### PR DESCRIPTION
**Pull request by adamrobinson361**
Background

To resolve #2

QA Required

@calstaff91 , @ZacharyWaller - can you both take a read through the preface and chapter 1 and provide feedback via this pull request on how you think it should be changed.

Note: @drmarkc13 added for proof reading too

### CODE REVIEW

File: index.Rmd [hunk 1]
```diff
@@ -9,4 +9,24 @@ link-citations: yes
...
+
+It is in the context of the Department for Education Corporate environment and focusses on using Git and Visual Studio Team Services (the departments existing code version control tool of choice). 
+
+R and R Studio are used as code base we are version controlling and IDE and the examples will be a mix of using R studio and the command line. 
+
+## Ways of Working
+
+On a high level this resource is advocating a new way of working for analytical projets which can be summarised as follows:
+
+1. **Plan your workload up front** - This will help you not only better understad the project but write less, better code to get the job done.
```
Comment (ZacharyWaller) [inline index.Rmd]: "understad the project" should be "understand the project"

**Comment by drmarkc13** [comment]
You getting your own back Zach? ☺

Cheers mate!

Mark.

File: index.Rmd (hunk 1 above)
Comment (adamrobinson361) [reply index.Rmd]: I'll fix that cheers

Comment (adamrobinson361) [reply index.Rmd]: Done - Actually a few more typos that got picked up with a spell check

**Pull request merged**